.github/workflows/python-build-artifacts.yml

```diff
@@ -18,6 +18,12 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Upload AaC models
+        uses: actions/upload-artifact@v2
```

Comment:
When you say [line 53 of `.github/workflows/python-tests.yaml`](https://github.com/jondavid-black/AaC/blob/200ef320455f0d460c8d2b9b70ef02f7cc1781f5/.github/workflows/python-tests.yml#L53) do you actually mean to put it in this stage?

https://github.com/jondavid-black/AaC/blob/200ef320455f0d460c8d2b9b70ef02f7cc1781f5/.github/workflows/python-tests.yml#L69-L85

Reply:
I can add that to the `Setup OS Test Matrix` but it'll also require checking out the repo if that's what you mean. Adding it to the `Python ${{ matrix.py_version }} Tests` section in `python-tests.yaml` would be an issue because it won't necessarily be uploaded before it needs to be downloaded unless we introduce a dependency between those jobs which will make the pipeline take even longer.